invenio_communities/members/resources/resource.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ignore for now until resources are addressed in future PR.

Reply:
Comment: To merge just make sure these files are cleaned out if not done